### PR TITLE
Edge network extensions

### PR DESCRIPTION
**Pull request by calebk1**
https://jira.corp.adobe.com/browse/MOB-16032

### CODE REVIEW

File: src/pages/documentation/edge-network-extensions/validation.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Validation
+
+The Adobe Experience Platform Edge extension is integrated with Adobe Assurance mobile extension to allow for request-response events inspection and provides rich verbose logging that can be used when verifying your mobile implementation.
+
+## Adobe Experience Platform SDK Logging
+
+The Adobe Experience Platform Mobile SDK and the AEP Edge extension allow for detailed debugging in debug or test environments when using Verbose or Debug Log level. To enable this and see the log messages in your IDE set the verbose/trace mode for [MobileCore logging](../getting-started/enable-debug-logging.md#debug-logging).
+
+## Inspect Experience events with Project Griffon
```
Comment (cacheung) [inline src/pages/documentation/edge-network-extensions/validation.md:9]: Inspect Experience events with Adobe Experience Platform Assurance

File: src/pages/documentation/edge-network-extensions/validation.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+While the log messages can be quite helpful for an application developer, inspecting the events in the [Project Griffon UI](https://experience.adobe.com/griffon/) is even easier. In just a few steps, you can view the timestamp ordered list of all the events generated by the AEP Edge mobile extension, along with the confirmation that the request was successfully processed by the AEP Edge Network.
```
Comment (cacheung) [inline src/pages/documentation/edge-network-extensions/validation.md:11]: Change all griffon to Assurance in this page.

File: src/pages/documentation/edge-network-extensions/validation.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Validation
+
+The Adobe Experience Platform Edge extension is integrated with Adobe Assurance mobile extension to allow for request-response events inspection and provides rich verbose logging that can be used when verifying your mobile implementation.
+
+## Adobe Experience Platform SDK Logging
+
+The Adobe Experience Platform Mobile SDK and the AEP Edge extension allow for detailed debugging in debug or test environments when using Verbose or Debug Log level. To enable this and see the log messages in your IDE set the verbose/trace mode for [MobileCore logging](../getting-started/enable-debug-logging.md#debug-logging).
```
Comment (cacheung) [inline src/pages/documentation/edge-network-extensions/validation.md:7]: Not sure if logging link is correct or not.

**Review by cacheung — COMMENTED** [review]
Have some questions about the links.

File: src/pages/documentation/edge-network-extensions/tabs/api-reference/send-event/android.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+#### Java
+
+**Syntax**
+
+```java
+public static void sendEvent(final ExperienceEvent experienceEvent, final EdgeCallback callback);
+```
+* _experienceEvent_ - the XDM [Experience Event](#experienceevent) to be sent to Adobe Experience Platform Edge Network
```
[the remark this answers is not in the document]
Comment (calebk1) [reply src/pages/documentation/edge-network-extensions/tabs/api-reference/send-event/android.md:8]: Yeah, it should link properly. I'll do some proper testing once it's merged in, though.

**Pull request merged**